# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/util-linux/linux-compiler_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./partx/addpart.c.orig	2004-12-30 18:27:39.994411784 +0100
+++ ./partx/addpart.c	2004-12-30 18:27:49.601951216 +0100
@@ -3,6 +3,7 @@
 #include <fcntl.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
+#include <linux/compiler.h>
 #include <linux/blkpg.h>
 
 int
--- ./partx/delpart.c.orig	2004-12-30 18:28:55.711900976 +0100
+++ ./partx/delpart.c	2004-12-30 18:29:03.194763408 +0100
@@ -3,6 +3,7 @@
 #include <fcntl.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
+#include <linux/compiler.h>
 #include <linux/blkpg.h>
 
 int
--- ./partx/partx.c.orig	2004-12-30 18:29:10.607636480 +0100
+++ ./partx/partx.c	2004-12-30 18:29:21.124037744 +0100
@@ -37,6 +37,7 @@
 #include <getopt.h>
 #include <unistd.h>
 #include <sys/ioctl.h>
+#include <linux/compiler.h>
 #include <linux/hdreg.h>        /* HDIO_GETGEO */
 #include <linux/blkpg.h>
 #define BLKGETSIZE _IO(0x12,96)    /* return device size */
